src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -112,6 +113,18 @@ function createPaymentRequest(args) {
       args.addTaxPercentage = true;
       if(paymentRequest.paymentMethod.type.indexOf('klarna') > -1){
         args.addTaxPercentage = false;
+        const address = order.getBillingAddress();
+        const otherDeliveryAddress = {
+          shipping_method: order.getDefaultShipment().shippingMethod.displayName,
+          shipping_type: order.getDefaultShipment().shippingMethod.description,
+          first_name: address.firstName,
+          last_name: address.lastName,
+          street_address: `${address.address1} ${address.address2}`,
+          postal_code: address.postalCode,
+          city: address.city,
+          country: address.countryCode.value,
+        }
+        paymentRequest.additionalData['openinvoicedata.merchantData'] = StringUtils.encodeBase64(JSON.stringify(otherDeliveryAddress));
```

Comment:
LGTM! maybe we can add a comment here to explain why the encoding is needed

Reply:
What if there is no shippingMethod, for example with pick up? It would throw an error here.

Reply:
I have not been able to reproduce. The only store pickup shipping method I can create is one that costs 0. I think there should always be a shipping method.